backend/src/main/migrations/20211004004829-create-task.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Task', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      description: {
+        type: Sequelize.STRING
+      },
+      status: {
+        type: Sequelize.ENUM('NEW', 'ACTIVE', 'RESOLVED', 'CLOSED', 'REMOVED'),
+        defaultValue: 'NEW'
+      },
+      deadlineDate: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      createdDate: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      modifiedDate: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      projectId: {
+        allowNull: false,
+        type: Sequelize.BIGINT,
+        onDelete: 'CASCADE',
```

Comment:
@nasaku898  Is Cascade appropriate here?

Reply:
I don't think you should cascade. Because i believe the interaction right now would be if you delete a task, it will delete the project too.

Reply:
i retract my statement. I think you can keep it.